app/script/auth/page/Login.js

```diff
@@ -0,0 +1,211 @@
+                      value={email}
+                      autoComplete="section-login email"
+                      placeholder={_(loginStrings.emailPlaceholder)}
+                      onKeyDown={event => {
+                        if (event.key === 'Enter') {
+                          this.inputs.password.focus();
+                        }
+                      }}
+                      maxLength="128"
+                      type="text"
+                      required
+                      data-uie-name="enter-email"
+                    />
+                    <InputSubmitCombo>
+                      <Input
+                        name="password"
+                        onChange={event =>
+                          this.setState({
+                            password: event.target.value,
+                            validInputs: {...validInputs, password: true},
+                          })
+                        }
+                        innerRef={node => (this.inputs.password = node)}
+                        markInvalid={!validInputs.password}
+                        value={password}
+                        autoComplete="section-login password"
+                        type="password"
+                        placeholder={_(loginStrings.passwordPlaceholder)}
+                        maxLength="1024"
+                        minLength="8"
+                        pattern=".{8,1024}"
+                        required
+                        data-uie-name="enter-password"
+                      />
+                      <RoundIconButton
+                        disabled={!email || !password}
+                        type="submit"
+                        formNoValidate
+                        onClick={this.handleSubmit}
+                        data-uie-name="do-next"
+                      />
+                    </InputSubmitCombo>
+                  </InputBlock>
+                  {validationErrors.length ? (
+                    parseValidationErrors(validationErrors)
+                  ) : loginError ? (
+                    <ErrorMessage data-uie-name="error-message">{parseError(loginError)}</ErrorMessage>
+                  ) : null}
+                  <Checkbox
+                    onChange={event => this.setState({persist: event.target.checked})}
+                    checked={persist}
+                    data-uie-name="check-persist"
+                    style={{justifyContent: 'center'}}
+                  >
+                    <CheckboxLabel>{_(loginStrings.publicComputer)}</CheckboxLabel>
+                  </Checkbox>
+                </Form>
+              </div>
+              <Columns>
+                <Column>
+                  <Link href={ROUTE.PHONE_LOGIN + window.location.search}>{_(loginStrings.forgotPassword)}</Link>
+                </Column>
+                <Column>
+                  <Link href={ROUTE.PASSWORD_RESET + window.location.search}>{_(loginStrings.phoneLogin)}</Link>
```

Comment:
These two look swapped. Password reset also is handled on account.wire.com. This URL won't work.